Jenkinsfile.dev

```diff
@@ -67,43 +53,69 @@ if(hasRepoChanged){
               type: "button",
               text: "View Build Logs",
               style:"danger",           
-              url: "${currentBuild.absoluteUrl}/console"
+              APP_URL: "${currentBuild.absoluteAPP_URL}/console"
```

Comment:
Question: Is `${currentBuild.absoluteAPP_URL}` correct? It got changed from `${currentBuild.absoluteUrl}` and seems like a text replacement for "url" gone wrong...

Reply:
Yes, good catch. I keep doing this search/replace fails... -.-